site/_docs/history.md

```diff
@@ -28,23 +28,50 @@ For a full list of releases, see
 Downloads are available on the
 [downloads page]({{ site.baseurl }}/downloads/).
 
-<!--
-## <a href="https://github.com/apache/calcite/releases/tag/calcite-1.34.0">1.34.0</a> / XXXX-XX-XX
+## <a href="https://github.com/apache/calcite/releases/tag/calcite-1.34.0">1.34.0</a> / 2023-03-XX
```

Comment:
can you add a commented-out section for 1.35

Reply:
Fixed in https://github.com/apache/calcite/pull/3103/commits/88c075dbbd36b164b521f417f2eade6799a3b569